Upgraded Grace reference to pre-release.

Lex45x opened the pull request:
Upgraded GraceServiceProvider and GraceServiceScope to be IAsyncDisposable

Lex45x committed:
Upgraded Grace reference to pre-release.
Upgraded GraceServiceProvider and GraceServiceScope to be IAsyncDisposable

ipjohnson commented on src/Grace.DependencyInjection.Extensions/Grace.DependencyInjection.Extensions.csproj.user:
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
I don't think we need this file

Lex45x replied:
Ah, my bad. Did not noticed this file.

ipjohnson reviewed (commented):
I think if you bump the version to 8.0.0-beta and remove the extra file this is good to go.

Lex45x committed:
Removed generated .user file.
Updated AppVeyor settings to build package with pre-release version (8.0.0-beta).

Lex45x commented:
Done!

ipjohnson commented:
This is released
